alpa/global_env.py

```diff
@@ -8,7 +8,7 @@ class GlobalConfig:
     def __init__(self):
         ########## Options of device mesh ##########
         self.xla_client_mem_fraction = float(
-            os.environ.get("XLA_PYTHON_CLIENT_MEM_FRACTION", 0.9))
+            os.environ.get("XLA_PYTHON_CLIENT_MEM_FRACTION", 0.7))
```

Comment:
0.9 is the default value in Jax and it works well for existing code. Changing it to 0.7 will make a lot of existing code OOM.
You can change this value in your own script for specific use cases.